packages/peregrine/lib/store/reducers/checkout.js

```diff
@@ -166,7 +177,21 @@ const reducerMap = {
             orderError: payload
         };
     },
-    [actions.reset]: () => initialState
+    [actions.reset]: state => {
+        const preservedState = {};
+
+        // Avoid refetching countries.
+        if (state.countries) {
+            preservedState.countries = {
+                ...state.countries
+            };
+        }
+
+        return {
+            ...initialState,
+            preservedState
```

Comment:
Having a `preservedState` property strikes me as odd. Did you mean to spread `preservedState` onto the return value instead?

Reply:
OOps

Reply:
```suggestion
            ...preservedState
```